pkg/handler/pullrequest/handler.go

```diff
@@ -72,10 +72,10 @@ func openOrSync(gitRepo *git.Git, pr *github.PullRequestPayload, gh ghclient.GH)
 	}
 
 	readyToReviewMsg := ""
-	if config != nil && config.LabelApproved != nil {
+	if config.HasLabelApproved() {
```

Comment:
```suggestion
	if config.LabelApproved() != nil {
```

Seems simpler to me this way (you can get rid of `HasLabelApproved()`)

Reply:
Yeah, it doesn't provide a lot of value. The patch is about removing pointers exposed to the user, but in the end we need to return the pointer for this one..